setup-env.template

```diff
@@ -8,8 +8,8 @@
 
 export ACCOUNT_ID=12345678
 export CENTRAL_ECR_ACCOUNT_ID=12345678
-export USE_CENTERAL_ECR=<true | false>
```

Comment:
:+1:

Reply:
I like that too,  it will clean up some shitty code in my "fix CI" PR.